var/spack/repos/builtin/packages/fastjet/package.py

```diff
@@ -61,6 +61,15 @@ class Fastjet(AutotoolsPackage):
 
     variant("shared", default=True, description="Builds a shared version of the library")
     variant("auto-ptr", default=False, description="Use auto_ptr")
+    variant(
+        "limited-thread-safety",
+        default=True,
+        when="@3.4.0:",
+        description="Enables limited thread safety",
+    )
+    variant(
+        "thread-safety", default=False, when="@3.4.0:", description="Enables full thread safety"
```

Comment:
What happens if both variants are set? Does this need a `conflict`?

Reply:
In that case the full thread safety is enabled.

Reply:
What I mean is that these two binary variants set up four possible states, but control a three-state system. That's not ideal, so we need to disallow one state or use a three-valued variant.

Reply:
I see, now there is multi-valued variant with three possible values "none", "limited", "full".